go.mod

```diff
@@ -245,11 +245,14 @@ require (
 	github.com/blugelabs/bluge v0.1.9
 	github.com/blugelabs/bluge_segment_api v0.2.0
 	github.com/dlmiddlecote/sqlstats v1.0.2
+	github.com/drone/drone-cli v1.5.0
 	github.com/getkin/kin-openapi v0.94.0
 	github.com/golang-migrate/migrate/v4 v4.7.0
 	github.com/google/go-github/v45 v45.2.0
+	github.com/grafana/build-pipeline v1.0.1
```

Comment:
Hmm, why do we need that? 🤔

Reply:
whoops that was auto imported. ran go mod tidy to resolve